lambdas/preprocessing/src/lambda_function.py

```diff
@@ -1,28 +1,75 @@
+import os
+from typing import Any, Dict, Optional
+from uuid import uuid4, UUID
+
+import geopandas as gpd
+import pandas as pd
+import tempfile
 from aws_xray_sdk.core import patch, xray_recorder
+from shapely.wkb import dumps as wkb_dumps
 
+from raster_analysis.boto import s3_client
 from raster_analysis.exceptions import QueryParseException
 from raster_analysis.globals import LOGGER
 
 patch(["boto3"])
 
+# FIXME: Get these from env
+BUCKET = "gfw-pipelines-test"
+REGION = "us-east-1"
```

Comment:
OK for now, but don't forget to fix these lines before we check into staging (especially line 18).